Assets/Scripts/ScreenShotScript.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+public class ScreenShotScript : MonoBehaviour
+{
+
+	public AudioSource shutterAudio;
+	public RawImage screenShot;
+	private Texture2D screenShotTexture;
+	private string lastScreenShotPath;
+	
+	// Use this for initialization
+	void Start ()
+	{
+		screenShotTexture = new Texture2D (100, 100);
+		screenShot.texture = screenShotTexture;
+	}
+	
+	// Update is called once per frame
+	void Update ()
+	{
+		
+	}
+
+	void OnGUI ()
+	{
+		Event e = Event.current;
+		
+		if (e.type == EventType.KeyDown && e.keyCode == KeyCode.F) {
+			string current_time = System.DateTime.Now.ToString ().Replace ("/", "_").Replace (":", "_");
+			lastScreenShotPath = Application.temporaryCachePath + "/" + current_time + ".png";
```

Comment:
<img width="857" alt="screen shot 2015-12-03 at 9 08 55 pm" src="https://cloud.githubusercontent.com/assets/7931364/11560208/3b29d8c6-9a02-11e5-92ad-eb98cc71e65e.png">

On my machine this is how it's like.